namespace py gen_py.utils
namespace java gen_java.utils

enum ServiceErrorCode {
    Nop = 0,
    ZeroDiv = 1
}